### PR TITLE
change the Filter parameter from *http.Request to *gin.Context #3070

### PR DESCRIPTION
**Pull request by rehanpfmr**
change #3070

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+### Changed
+
+- Change the Filter parameter from *http.Request to *gin.Context in `go.opentelemetry.io/contrib/instrumentation/github.com/gin-gonic/gin/otelgin` (#3070)
```
Comment (dmathieu) [inline CHANGELOG.md:13]:
```suggestion
- Change the Filter parameter from *http.Request to *gin.Context in `go.opentelemetry.io/contrib/instrumentation/github.com/gin-gonic/gin/otelgin` (#4375 )
```

**Review by dmathieu — COMMENTED** [review]
Please bring back `instrgen/docs/flow.png`.

**Comment by dmathieu** [comment]
Note that you don't need to close/reopen PRs for every change made. If you push a new commit (or amend an existing commit) to the same branch on your fork, the PR will be updated.

**Comment by rehanpfmr** [comment]
I had unexplainable issue fixing GO package changes, So i had to just close and reopen new one. I hope I will no longer have to do it again.

**Comment by dmathieu** [comment]
The `instrgen/docs/flow.png` file is currently deleted in your PR. Could you bring it back?

**Comment by rehanpfmr** [comment]
@dmathieu the instrgen/docs/flow.png is already there. However it shows as "Unable to render rich display Invalid image source"
https://github.com/open-telemetry/opentelemetry-go-contrib/pull/4375/commits/04e9f5bde317be2b328613368aad996474424ef7

**Comment by rehanpfmr** [comment]
please help fix the image

**Comment by dmathieu** [comment]
Yes, it's already on the main branch of the repository. But if you look at the diff for your PR (https://github.com/open-telemetry/opentelemetry-go-contrib/pull/4375/files), you will see it changes when it shouldn't.

**Review by hanyuancheung — APPROVED** [review]

**Comment by rehanpfmr** [comment]
Can this be merged?

**Comment by pellared** [comment]
> in branch of the repository. But if you look at the diff for your PR (https://github.com/open-telemetry/opentelemetry-go-contrib/pull/4375/files), you will see it chan

This issue is not addressed.

**Comment by pellared** [comment]
Superseded by https://github.com/open-telemetry/opentelemetry-go-contrib/pull/4444

**Pull request closed without merging**